client/main.lua

```diff
@@ -354,68 +400,85 @@ RegisterNetEvent('qb-diving:client:UseGear', function()
         end
     elseif iswearingsuit == true then
         gearAnim()
-        QBCore.Functions.Progressbar("remove_gear", Lang:t("info.pullout_suit"), 5000, false, true, {}, {}, {}, {},
-            function() -- Done
-                SetEnableScuba(ped, false)
-                SetPedMaxTimeUnderwater(ped, 50.00)
-                currentGear.enabled = false
-                ClearPedTasks(ped)
-                deleteGear()
-                QBCore.Functions.Notify(Lang:t("success.took_out"))
-                TriggerServerEvent("InteractSound_SV:PlayOnSource", nil, 0.25)
-                iswearingsuit = false
-                oxgenlevell = oxgenlevell
-            end)
+
+        QBCore.Functions.Progressbar("remove_gear", Lang:t("info.pullout_suit"), 5000, false, true, {}, {}, {}, {}, function() -- Done
```

Comment:
```suggestion
        QBCore.Functions.Progressbar("remove_gear", Lang:t("info.pullout_suit"), 5000, false, true, {}, {}, {}, {}, function()
```
Comments that should not need to be there.